### ISSUE TITLE
"Server with name X is not defined"

### ISSUE THREAD

**Issue by justcharlie** [Taxel/PlexTraktSync#1988]
### Confirmation

- [X] I have read the [README.md](https://github.com/Taxel/PlexTraktSync#readme) on the project homepage
- [X] I have checked if identical issue already exists
- [X] I have tried downgrading to find version that can be used as a workaround

### The problem

After successfully authenticating I get this error in plextraktsync which is running on Unraid. I searched and see that others have experienced this also but I couldn't find a solution. Thanks for any help you can offer.

### Steps to reproduce the behavior

Authenticate Trakt and run for the first time

### Expected behavior

It's supposed to scan my library

### Install method

docker-compose

### Version

0.31.0

### Python Version

3.12.4

### Plex Server Version

1.40.3.855-fef15d30c

### Operating System and Version

Unraid latest

**Comment by 2fersen** [comment]
I had the same problem.

The problem was the servers.yml in the **config folder**. After setup with docker there were two servers stored in it. One was called "default" and the other was mine, I just had to delete the default one.

I also deleted all the trashy or old server urls and just left one with the IP address of my server.


The file should look like this:

```
servers:
  Example1:
    token: ~
    urls:
      - http://localhost:32400
    config:
```
